Fluent/Simulation.ipynb

```diff
@@ -55,51 +55,33 @@
-   "\n"
-   "- In order to use the report function accurately during post-processing, the reference values should be computed from the inlet. This defines the reference values from the working fluid and domain properties. User matches the values: `/define/reference-values/compute-from/inlet`."
-   
-   
+    "<font color='red'>**3. Setup Boundary and Zone Conditions:** (Add the required bullet points with details explanations. Keep the title but remove the color)</font>\n",
+    "\n",
+    "* Explain the boundary and zone conditions chosen and defined within your CFD domain. \n",
+    "* Link your choices, with reasonable arguments, to the discussed physical principals and fundamentals discussed in Physics section.\n",
+    "* Discuss potential changes in the boundary condition if applicable for slightly different case studies.\n",
     "\n",
     "<br>\n",
-    "**4. Setup Solution methods:** \n",
+    "<font color='red'>**4. Setup Solution methods:** (Add the required bullet points with details explanations. Keep the title but remove the color)</font>\n",
     "\n",
-    "- In this step, it is highly recommended to use the default options and settings, unless based on the physics of the problem the user is aware of any specific choices. Upon non-smooth convergence and potential divergence of the CFD simulation user can modify and examine the various solution methods. To modify the solution methods and controls use the following commands respectively: \n"
-    "\n"
-    "`solve/set/discretization-schem` \n"
-    "\n"
-    "`solve/set/under-relaxation` \n"
-    
+    "* Explain your choices for numerical scheme and models to solve the flow field governing equations numerically.\n",
+    "*  Link your choices, with reasonable arguments, to the discussed physical principals and fundamentals discussed in Physics section.\n",
     "\n",
     "Now that all boundary conditions and settings for the CFD simulation are defined. User can **initialize** the solution through an educated guess to start the iteration process: `/solve/initialize/compute-defaults/velocity-inlet`\n",
-    "Solution initialization would incept the flow field variables, such as velocity and pressure, based on the defined values by user. For the current problem the CFD domain is recommended to be initialized using values of velocity and pressure at the inlet.\n",
+    "Solution initialization would incept the flow field variables, such as velocity and pressure, based on the defined values by user. For the current problem the CFD domain is recommended to be initialize by values of velocity and pressure at the inlet.\n",
     "\n",
```

Comment:
Great explanation on the time steps. If possible it would be great if you can elaborate more on the trial and error process to decide about the time step and number of iterations. Either generally talk about it or specifically explain the process that you went through (if any).